Zilon.Core/Zilon.Core.Tests/StaticObjectModules/TestCaseDataSources/DepositDurabilityModuleTestCaseDataSource.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+
+using NUnit.Framework;
+
+namespace Zilon.Core.Tests.StaticObjectModules.TestCaseDataSources
+{
+    public static class DepositDurabilityModuleTestCaseDataSource
+    {
+        public static IEnumerable DestroyTestCases
+        {
+            get
+            {
+                // StockValue = 1;
+                // DamagePerMineUnit = 10;
+                // damages = new int[] { 6, 4 };
+                yield return new TestCaseData(1, 10, new int[] { 6, 4 });
+                yield return new TestCaseData(2, 10, new int[] { 6, 4, 5, 5 });
+                yield return new TestCaseData(1, 1, new int[] { 2 });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5427118)